_layouts/newsletter.html

```diff
@@ -3,18 +3,22 @@
 --- 
 <link rel="stylesheet" href="/assets/css/main.css">
 
-{% include functions/localization-finder.md %}
+{% assign baseurl = page.permalink | remove_first: "/" | remove_first: page.lang %}
 
 <article class="newsletter">
   <header class="post-header">
-    {% if localizations.size > 0 %}
-      <div class="localization">
-        <a href="{{ '/en' | append:baseurl }}">en</a>
-        {% for locale in localizations %}
-          | <a href="{{ locale.url }}">{{locale.lang}}</a>
-        {% endfor %}
-      </div>
-    {% endif %}
+    <div class="localization">
+      <a href="{{ '/en' | append:baseurl }}">en</a>
+      {% for lang in site.languages %}
+        {% assign localization = "/" | append: lang | append:baseurl %}
+        {% assign locale = site.posts | where:"permalink", localization %}
+        {% if locale.size == 0 %}
+          | <span title="No {{lang}} translation currently available">{{ lang }}</span>
```

Comment:
Is there a way to display the full name of the language in this pop up, eg:

"No Spanish translation currently available"

instead of:

"No es translation currently available"

Reply:
@bitschmidty If you put a dict in `_config.yml` like:

    langs:
      en: English
      es: Spanish
      etc...

You should be able to do something like `"No {{site.langs[lang]}} translation currently available"`